tools/peptideshaker/datatypes_conf.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<datatypes>
+  <registration display_path="display_applications">
+    <datatype extension="par" type="galaxy.datatypes.text:Json" display_in_upload="true" subclass="true" mimetype="application/json">
```

Comment:
This should go upstream into Galaxy. Local datatypes should not be used anymore.
As you subclass JSON here you don't need to specify the parser.
I would also rename this to maybe `searchgui_par`